.github/actions/fling-test/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: Test-flinging
+description: Submit a job YAML file to a Testflinger server
+inputs:
+  job:
```

Comment:
Looking at the example I see why providing a `job` option is necessary (the very least since the `...-data` contains also the commands).

However, we should _also_ accept the job as inline input, and then we'll do breaking changes until that becomes practical to use.

Perhaps `job-path` vs `job`?

Reply:
Initially `job` was supposed to be the inline text for the job YAML and then I was practically _forced_ to switch to accepting a file instead, which turned out to be clean and practical. I ran into all sorts of issues when I tried to pass the corresponding text of the job yaml between workflow steps. I came to the conclusion that passing long multiline strings between steps (i.e. saving the text as the output of one step and then retrieving it from one of the following steps) is not a good pattern. I will investigate some more, just to be sure.

Reply:
Please provide `job-path` and `job` or something to this effect. As noted above, as discussed offline: we'll do breaking changes in testflinger in follow-up until that becomes practical to use across a wide set of usecases (since it's better UX).